src/Raven.Server/Utils/MicrosoftLogging/SparrowLoggerWrapper.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Microsoft.Extensions.Logging;
+using Sparrow.Logging;
+
+namespace Raven.Server.Utils.MicrosoftLogging;
+
+public class SparrowLoggerWrapper<TLogger> : ILogger<TLogger>
+{
+    private readonly Logger _sparrowLogger;
+
+    public SparrowLoggerWrapper(Logger sparrowLogger)
+    {
+        _sparrowLogger = sparrowLogger;
+    }
+
+    public IDisposable BeginScope<TState>(TState state) where TState : notnull
+    {
+        return NullScope.Instance;
+    }
+
+    public bool IsEnabled(LogLevel logLevel) => true;
+
+    public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+    {
+        if (formatter == null)
+            throw new ArgumentNullException(nameof(formatter));
+        
+        var logLine = formatter(state, null);
```

Comment:
shouldn't we pass the exception here?

Reply:
I passed it to our logger.
I don't want to have the exception twice